plugins/runner-postcss/__tests__/default.test.ts

```diff
@@ -74,5 +95,38 @@ test('Resolves postcss @import', async () => {
     ]
   });
 
+  const [hash] = extractHashes(result, 'mockHash');
+
   expect(result).toMatchSnapshot();
+
+  if (os.platform() === 'win32') {
+    expect(hash).toBe('b2fa5e79e89879036728c19a038eafac');
+  } else {
+    expect(hash).toBe('8de170ce8b6d1c1aeb78d959f4faa0cf');
+  }
 });
+
+function extractHashes(
+  result: ExecuteRunner2Result,
```

Comment:
probably you could use `ReturnType<typeof executeRunner2>`

Reply:
I tried but it didn't work as expected since executeRunner2 returns a Promise, and here I am using the wrapped value of the Promise.  I found some workaround using conditional type inference and it seems to work though it's a bit obscure `(Unpromise<T>)`.  Do you prefer this approach?

Reply:
true, maybe then you can use unpromise to define the ExecuteRunner2Result type so it doesn't need to be kept in sync manually with what function returns, what do you think?

Reply:
Hmm, I think I'm not understanding.  

I am doing `UnPromise<ReturnType<typeof executeRunner2>>` now and it works and I deleted the `ExecuteRunner2Result` so that type inference is used instead and it's working.  This way, there is nothing to keep in sync. 
 Do you want me to put the explicit type back in again?

Reply:
Sorry, I meant that in order to not define ad-hoc type helpers which are universal(like unpromise) in that particular test file, you could put them to where executeRunner2 is defined and use to create ExecuteRunner2Result type instead of defining it's detailed structure. But I think we can leave it as it's now as postcss test is the only place using ExecuteRunner2Result. But if you're foreseeing usage of this type in other tests, maybe it makes sense to put ExecuteRunner2Result next to executeRunner2 function and define it through the inference

Reply:
Ahh yes, what you're saying makes perfect sense.  Also, by doing it that way, external TypeScript users of executeRunner2 can also now more easily define utility functions on the result as they see fit.  Changed it as suggested.  What do you think?

Also, I squashed and did a bit more manual testing of runner-ts to triple check and everything seems ok.  I think we're good to merge now.